assets/js/projects.js

```diff
@@ -30,6 +30,23 @@ document.addEventListener('DOMContentLoaded', function() {
             // Card data goes here
             // Example:
             // category: 'consultancy', 'software development', 'support and training'],
+            {
+              category: ['support and training'],
+              status: 'past',
+              imageUrl: 'assets/img/projects/eurelectri.jpeg',
+              title: 'Comprehensive PyPSA Training Session',
+              subtitle: 'Equipping energy enthusiasts with essential skills for open energy modeling',
```

Comment:
Could it be a good idea to highlight that the audience is also professional in energy? [Though, the colleagues are for sure also energy modeling enthusiasts] Also, that has been a rather in-depth course, actually the most comprehensive one from those we held as far. That could be probably also worth mentioning. A possible revision:

```suggestion
              subtitle: 'Equipping energy professionals with advanced skills of open source energy modeling',
```